tracks/python/exercises/armstrong-numbers/mentoring.md

```diff
@@ -0,0 +1,64 @@
+# Mentoring
+
+## Problem and challenges
+
+The problem requires the student to sum each digit of an integer or _natural number_ raised to the number of digits' power.
+
+## Reasonable solutions
+
+This exercise can be solved in a fairly straightforward fashion:
+
+```python
+def is_armstrong(number):
+    digits = (int(d) for d in str(number))
+    count = len(str(number))
+    total = sum(digit ** count for digit in digits)
+    return number == total
+```
+
+This solution can also be written without the generator.
+This avoids the more complex generator type at the expense of a more complex sum line.
+
+```python
+def is_armstrong(number):
+    count = len(str(number))
+    total = sum(int(digit) ** count for digit in str(number))
+    return number == total
+```
+
+### Common Suggestions
+
+#### Generator Expressions
+
+```python
+    # With an intermediate list passed to sum.
+    total = sum([digit ** count for digit in digits])
+    
+    # With a generator expression passed to sum
+    total = sum(digit ** count for digit in digits)
+
+Students are often unaware of [generator expressions](https://www.python.org/dev/peps/pep-0289/) and that they can be passed directly to functions that accept iterables.
```

Comment:
If you want to do so now, reference links are preferred according to the Markdown Guide for Exercism.

Reply:
I'd prefer to defer that change and let it happen with the mass fix PR.

Reply:
I was unaware of a mass fix PR that would happen.  I will defer to that as well then.